src/plot.js

```diff
@@ -159,6 +159,11 @@ export function plot(options = {}) {
   context.className = className;
   context.projection = createProjection(options, subdimensions);
 
+  // A path generator for marks that want to draw GeoJSON.
+  context.path = function () {
+    return geoPath(this.projection ?? xyProjection(scales));
+  };
```

Comment:
```suggestion
  const projection = context.projection = createProjection(options, subdimensions);

  // A path generator for marks that want to draw GeoJSON.
  context.path = () => geoPath(projection ?? xyProjection(scales));
```

this is maybe more consistent?